runtime/makelib/targets.mk.ftl

```diff
@@ -597,7 +594,7 @@ ifndef UMA_NO_CLEAN_TARGET
 #
 clean:
 	$(RM) $(UMA_OBJECTS) $(UMA_BYPRODUCTS) $(TARGETS)
-	-$(RM) *.d $(UMA_OBJECTS:$(UMA_DOT_O)=.i)
+	-$(RM) *.d $(UMA_OBJECTS:$(UMA_DOT_O)=.i) $(UMA_OBJECTS:$(UMA_DOT_O)=.lst)
```

Comment:
Are `.lst` files ztpf specific?  If so the RM command should be only included on ztpf specs